### DISCUSSION TITLE
Add a thousands separator in sub-editor

### DISCUSSION THREAD

**Discussion by Hinny** [FakeFishGames/Barotrauma#9169] [Ideas and suggestions]
It would be a nice quality of life improvement to display  thousands separators in the game, especially in the sub-editor.

1234567 => 1,234,567 (US style) or 1 234 567 (Swedish style).

It is very hard to see at a glance which is a larger hull value, 12752211 or 1643287.

12,752,211 vs 1,643,287 and 12 752 211 vs 1 643 287 is much easier.

**Comment by MaaiG** [comment]
Yea, but suggestion to addition:
The game should take the "Digit grouping synbol" and the "Decimal symbol" from the system’s regional settings. [Replace the separator with the sign used by the system](https://github.com/Regalis11/Barotrauma/issues/7837#issuecomment-997215641).

**Comment by MateusJK15** [comment]
Would be a great QoL feature. Some countries uses 1.234.567 so the devs could add an option in the settings menu to change the format.

**Reply by MaaiG** [reply]
Lot of variants :)
And they can all be read from the settings of your operating system!
![Image 001_13-12 (1)](https://user-images.githubusercontent.com/93348499/170239488-67641cad-0c78-4695-981c-11411f801a6e.png)
